src/v2/components/ChannelPreview/hooks/useChannelPreviewDisplayName.ts

```diff
@@ -92,7 +92,14 @@ export const useChannelPreviewDisplayName = <
   const channelName = channel?.data?.name;
   const maxCharacterLength = characterLength || maxCharacterLengthDefault;
 
-  const [displayName, setDisplayName] = useState('');
+  const [displayName, setDisplayName] = useState(
+    getChannelPreviewDisplayName({
```

Comment:
Unrelated, but making sure here that initial value is computed as well, instead of empty string - which results into blank view at first.